api/gluster-create-volume-example.adoc

```diff
@@ -0,0 +1,131 @@
+// vim: tw=79
+= Gluster Volume
+
+The document describes the flow for creating a new Gluster volume using the
+Tendrl API.
+
+To begin with, we need to explore the available actions that you can take on a
+volume, to discover the actions we can use the actions API on the
+resource for the defined cluster.
+
+Sample Request
+
+----------
+GET /cluster/:cluster_id/volume/actions
```

Comment:
Yes, we know the type of the cluster with the :cluster_id in the URL, so we know what kind of actions and attributes are possible on that specific cluster.

There will also be a endpoint to find the resources which are possible on the cluster
`  GET /cluster/:cluster_id/actions `

To find the attributes to create a cluster we will have
` GET /cluster/attributes`

But the example is very specific to a volume, hence it only talks about the volume actions and attributes, when we talk about the cluster creation the other endpoints will be documented.